uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -23,7 +23,8 @@ class DrupalUserServiceTest extends CRM_HRCore_Test_BaseHeadlessTest {
   private $testContact;
 
   public function setUp() {
-    $this->testContact = ContactFabricator::fabricate();
+    CRM_Utils_API_ReloadOption::singleton();
+    $this->loadRequiredData();
```

Comment:
Do we need to clear anything created by these methods on the `tearDown()` method? Or the transaction rollback after each test is enough to keep things clear?